src/app/components/signin/signin.component.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]="form">
+    <h2>Sign in</h2>
+  
+    <label>Email</label><br><br>
+    <input type="email" formControlName="email"><br><br>
+    <label>Password</label><br><br>
+    <input type="password" formControlName="password"><br><br>
```

Comment:
too many `br`s
can we do without them?

Reply:
The br are essential since most of the forms elements are inline elements

Reply:
display flex or changing inline elements to display block could help reduce them